samples/css/samples.css

```diff
@@ -1,5 +1,5 @@
 /**
- * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
Please revert this change, it's not a subject of this issue. It's done in other PR, and your change here will cause merging conflicts.

Remove it by editing your commit history, as if the change never happened.